telegram/ext/jobqueue.py

```diff
@@ -469,6 +490,28 @@ def interval_seconds(self):
         else:
             return interval
 
+    @property
+    def next_t(self):
+        """
+        ::obj:`datetime.datetime`: Datetime for the next job execution.
+            Datetime is localized according to :attr:`tzinfo`.
+            If job is removed or already ran it equals to ``None``.
+
+        """
+        return datetime.datetime.fromtimestamp(self._next_t, self.tzinfo) if self._next_t else None
+
+    @next_t.setter
+    def next_t(self, next_t):
```

Comment:
Setting `next_t` should not be accessible to users. It is an internal implementation detail which we merely give the option to peek at.
Setting `next_t` will most like not have the affect a user would expect (it would probably do nothing and only change the value which can be "peeked" at).

So, what I'm suggesting is:
1. This shouldn't be a property setter.
2. Rename this function to: `def _set_next_t(self, next_t):`
3. Line 309 (part of the JobQueue code) will do:
```python
job._set_next_t(next_t)
```
Despite looking "ugly" (as `JobQueue` is accessing a "private" method of `Job`), it is not. The way I see it, `Job` is an implementation detail of `JobQueue` and as such, `JobQueue` is allowed to access such a "private" method.